elliottlib/cli/__main__.py

```diff
@@ -57,6 +57,9 @@
 from elliottlib.cli.verify_cvp_cli import verify_cvp_cli
 from elliottlib.cli.advisory_drop_cli import advisory_drop_cli
 from elliottlib.cli.verify_attached_operators_cli import verify_attached_operators_cli
+from elliottlib.cli.advisory_drop_cli import advisory_drop_cli
+from elliottlib.cli.verify_attached_operators_cli import verify_attached_operators_cli
```

Comment:
This line is a duplicate.
```suggestion
```

Reply:
*now* it should be removed ;D i rebased master again.